gremlin-go/driver/driverRemoteConnection.go

```diff
@@ -54,6 +54,7 @@ type DriverRemoteConnectionSettings struct {
 type DriverRemoteConnection struct {
 	client          *Client
 	spawnedSessions []*DriverRemoteConnection
+	closed          bool
```

Comment:
nit: `isClosed` is a bit better.

Reply:
compare with https://github.com/lyndonbauto/tinkerpop/blob/gremlin-go-ms5/gremlin-go/driver/resultSet.go#L53